pretty-printer.lua

```diff
@@ -148,11 +171,11 @@ function PrettyPrint:table(fields)
 		self[#self + 1] = self:_resetcolor()
 	else
 		self[#self + 1] = self:_color()
-		self[#self + 1] = " {\n"
+		self[#self + 1] = "{\n"
 		self[#self + 1] = self:_resetcolor()
 		self:_indent()
 		for k, v in pairs(fields) do
-			if not hidden_fields[k] then
+			if k ~= "kind" and not hidden_fields[k] then
```

Comment:
add kind to hidden_fields at the top instead of denying it here separately?

Reply:
also not sure we need to skip `kind` because record printers generated by the terms-generator already skip it so it'd only show up on something that isn't from the term system